Makefile

```diff
@@ -328,7 +327,7 @@ update-build: upload-build upload-lambdas
 			"ParameterKey=GitHubBranch,ParameterValue=${REPO_BRANCH}" \
 			"ParameterKey=GitHubToken,ParameterValue=$(shell aws ssm get-parameter --name /${OWNER}/${PROJECT}/build/REPO_TOKEN --output json --with-decryption | jq -r '.Parameter.Value')" \
 			"ParameterKey=ApplicationName,ParameterValue=${REPO}" \
-			"ParameterKey=Prefix,ParameterValue=${PREFIX}" \
+			"ParameterKey=Subdomain,ParameterValue=${SUBDOMAIN:-`echo ${REPO}`}" \
```

Comment:
any value to this pattern vs above for PREFIX = default to OWNER?

Reply:
It has to be per-build-stack.  The OWNER/PREFIX stuff was global.